delete_dashboards_by_text_in_title/delete_dashboards.py

```diff
@@ -0,0 +1,45 @@
+from datadog import initialize, api
+
+
+def delete_dashboards_by_text_in_title():
```

Comment:
I'd go as far as backing up the dashboards before deleting them or at least providing the option.  We could include something as simple as this:  https://github.com/DataDog/Miscellany/blob/462a02164acf2d061df0ac3fd3e8fbec31ba972b/Dogmover/dogmover.py#L75